src/components/GameForm.js

```diff
@@ -0,0 +1,60 @@
+import Button from './Button';
+import Input from './Input';
+import styled from 'styled-components';
+import { useState } from 'react';
+
+const initialFormData = {
+  nameOfGame: '',
+  playerNames: '',
+};
+
+export default function GameForm({ onCreateGame }) {
+  const [formData, setFormData] = useState(initialFormData);
+
+  return (
+    <FormGrid
```

Comment:
Maybe just `Form`, I don't care about the layout when I want to understand how it works?

Reply:
Yes, I agree.